lib/src/response/responder.rs

```diff
@@ -201,6 +201,16 @@ impl Responder<'static> for String {
     }
 }
 
+/// Returns a response with a fixed-size body
+/// containing the byte vec `self`. Always returns `Ok`.
+impl Responder<'static> for Vec<u8> {
```

Comment:
Because the `Responder` wants to return a `Result` with `rocket::Response`, so `Cursor<Vec<u8>>` won't fit that type definition.